tests/README.md

```diff
@@ -0,0 +1,33 @@
+## Frontend Tests 
+
+##Folder Structure
```

Comment:
need to add a space here for formatting to work properly

Reply:
fixed. Thanks!